src/index.js

```diff
@@ -1,29 +1,24 @@
-responseCount = 0
-currentQuestion = 0
 options = {
   url: 'data/quiz.json?' + Date.now()
 }
 
 $.ajax({
   url: options.url
-}).done(function(data) {
-  questions = data.questions
+}).done(function (data) {
+  var responseCount = 0
```

Comment:
Prefer let over var.
You could have left a unique declaration of these variables as destructuring.